macos/Runner/AppDelegate.swift

```diff
@@ -12,18 +12,9 @@ class AppDelegate: FlutterAppDelegate {
   }
 
   override func applicationDidFinishLaunching(_ aNotification: Notification) {
-      DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
-        let newWindow = SideFlutterWindow(engine: MainFlutterWindow.engine!)
-        newWindow.title = "New Window"
-        newWindow.isOpaque = false
-        newWindow.center()
-        newWindow.isMovableByWindowBackground = true
-        newWindow.isReleasedWhenClosed = false
-        newWindow.backgroundColor = NSColor(calibratedHue: 0, saturation: 1.0, brightness: 0, alpha: 0.7)
-        DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) {
-          newWindow.activate()
-          newWindow.makeKeyAndOrderFront(nil)
-        }
-      }
+    let newWindow = SideFlutterWindow(engine: MainFlutterWindow.engine!)
+    DispatchQueue.main.asyncAfter(deadline: .now() + 4.0) {
+      let newWindow = SideFlutterWindow(engine: MainFlutterWindow.engine!)
```

Comment:
The compiler is also warning me that this variable and the one two lines above is unused.

Reply:
lol, if I don't assign it to a variable, it complains that the result of the initializer is unused...

Reply:
Not sure how to resolve this one then, but you can reproduce it by doing a `flutter clean` before doing a `flutter build`.